### ISSUE TITLE
related should pull xdata

### ISSUE THREAD

**Issue by bastien-roucaries** [plk/biblatex#1025]
related entry should pull xdata (i suppose it is a bug in biber)

```
\documentclass{article}

\begin{filecontents*}{minimal.bib}
@xdata{ClassicalElectrodynamics-schwinger:1998-2018,
  title =	 {Classical Electrodynamics},
  series =	 {Advanced Book Program},
  language =	 {langenglish},
  langid =	 {english},
  pagetotal =    592,
  abstract =	 {This book is an outgrowth of lectures that Julian
                  Schwinger gave at UCLA in 1976, when, for the first
                  time in many years, he was asked to teach Classical
                  Electrodynamics. Of course, Schwinger had
                  contributed greatly to the development of the
                  subject, partly through his profound work on the
                  theory of waveguides and microwave cavities at the
                  MIT Radiation Laboratory during World War II. Only a
                  small part of that material was ever published, in
                  Discontinuities in Waveguides.1 After the war,
                  Schwinger played a key role in the perfection of the
                  synchrotron, and, especially, was largely
                  responsible for the theory of synchrotron
                  radiation.2 His first course at Harvard in 1946,
                  Applied Science 33, was on advanced applications of
                  electromagnetic theory, particularly to
                  waveguides. (We have incorporated four problems from
                  the final exam to that course in the present
                  volume.) Schwinger's discoveries in classical
                  electrodynamics led directly to his solution of the
                  difficulties in quantum electrodynamics a few years
                  later.}
}

@xdata{author,
  author =	 {Julian Schwinger and De{R}aad, Jr, Lester L. and
                  Milton, Kimball A. and Tsai, Wu-Yang},
}
              
@book{ClassicalElectrodynamics-schwinger:1998,
  xdata =	 {ClassicalElectrodynamics-schwinger:1998-2018,author},
  isbn =	 {0-7382-0056-5},
  publisher =	 {Perseus Books},
  location =	 {Reading, {MA}, {USA}},
  date =	 {1998},
}

@book{ClassicalElectrodynamics-schwinger:2018,
  xdata =	 "ClassicalElectrodynamics-schwinger:1998-2018",
  author =	 {Jul Schwinger and De{R}aad, Jr, Lester L. and
                  Milton, Kimball A. and Tsai, Wu-Yang},
  isbn =	 {978-0-7382-0056-9},
  related =      {ClassicalElectrodynamics-schwinger:1998},
  relatedtype =  {reprintof},
  publisher =	 "{CRC} Press",
  location =	 {Boca Raton, {FL}, {USA}},
  doi =          {10.1201/9780429503542},
  date =	 {2018},
}
\end{filecontents*}

\usepackage[backend=biber,style=numeric-comp]{biblatex}
\addbibresource{minimal.bib}
\begin{document}
\cite{ClassicalElectrodynamics-schwinger:2018}
\printbibliography
\end{document}
```
The bbl file show the problem no author pulled by xdata.
```
\entry{b3f8328fdd411252c94e521776693161}{book}{skipbib=true,skipbiblist=true,skiplab=true}
      \list{location}{1}{%
        {Reading, {MA}, {USA}}%
      }
      \list{publisher}{1}{%
        {Perseus Books}%
      }
      \field{sortinit}{1}
      \field{sortinithash}{50c6687d7fc80f50136d75228e3c59ba}
      \field{clonesourcekey}{ClassicalElectrodynamics-schwinger:1998}
      \field{isbn}{0-7382-0056-5}
      \field{year}{1998}
      \field{dateera}{ce}
    \endentry

```

**Comment by plk** [comment]
This was a bug, fixed in 2.15 DEV on Sourceforge.

**Comment by moewew** [comment]
`biblatex` 3.15 and Biber 2.15 are available in MikTeX and TeX live now.
